mds/json.py

```diff
@@ -51,24 +50,17 @@ def to_feature(shape, properties={}):
 
     Optionally give the Feature a :properties: dict.
     """
-    collection = to_feature_collection(shape)
-    feature = collection["features"][0]
-    feature["properties"] = properties
+    feature = mapping(shape)
 
-    # remove some unnecessary and redundant data
-    if "id" in feature:
-        del feature["id"]
-    if isinstance(shape, Point) and "bbox" in feature:
-        del feature["bbox"]
+    if isinstance(shape, Point):
+        feature["coordinates"] = list(feature["coordinates"])
```

Comment:
curious -- why is it important that the results are python `list`s rather than tuples?

Reply:
@johnclary want to chime in with any thoughts?

@ezheidtmann is there a performance or other consideration we should be aware of here?